### DISCUSSION TITLE
Custom template in Kubernetes

### DISCUSSION THREAD

**Discussion by SolasArd** [metalmatze/alertmanager-bot#185] [Help/Support]
Hello,

First of all, thank you for this great project! I was wondering how to customize the notifications template in the kubernetes environment because currently the alerts in Telegram aren't very informative:

```
🔥 FIRING 🔥
CpuUsage

Duration: -
```

I see that the notification template is in default.tmpl, what would be the best way to override it in Kubernetes?

**Comment by metalmatze** [comment]
Yes, you can customize the template. 
1. If you use the annotations `summary`, `description` or `message` these should show up with the default template without any extra work.
2. You can create a ConfigMap with your custom template and mount it into the container and then use the `--template.paths` flag to tell the bot where that config lives on the file system inside the container.

I hope that helps.
